client/README.md

```diff
@@ -114,6 +114,10 @@ func init() {
     if err != nil {
         panic(err)
     }
+   	proxies, err = toxiClient.Proxies()
+   	if err != nil {
+        panic(err)
+   	}
```

Comment:
This addition seems incorrect as `proxies` is assigned on line 108 as a result of the `toxiClient.Populate` call.

If this is meant as another example, similar to the example on line 122, I would be okay with this remaining but the indentation should be fixed, it should be commented out, and a comment added explaining what it does if its uncommented.